File: Entidades/PatenteEN.vb

Public Class PatenteEN
    Private _codpat As Integer
    Public Property CodPat() As Integer
        Get
            Return _codpat
        End Get
        Set(ByVal value As Integer)
            _codpat = value
        End Set
    End Property

    Private _descripcion As String
    Public Property Descripcion() As String
        Get
            Return _descripcion
        End Get
        Set(ByVal value As String)
            _descripcion = value
        End Set
    End Property
End Class

File: Entidades/PatenteEN.cs

namespace Entidades
{
    public class PatenteEN
    {
        private int _codpat;

        public int CodPat
        {
            get
            {
                return _codpat;
            }

            set
            {
                _codpat = value;
            }
        }

        private string _descripcion;

        public string Descripcion
        {
            get
            {
                return _descripcion;
            }

            set
            {
                _descripcion = value;
            }
        }
    }
}